NitroxServer/GameLogic/PlayerManager.cs

```diff
@@ -55,12 +56,24 @@ public List<Player> GetPlayers()
                     MultiplayerSessionReservationState rejectedState = MultiplayerSessionReservationState.Rejected | MultiplayerSessionReservationState.UniquePlayerNameConstraintViolated;
                     return new MultiplayerSessionReservation(correlationId, rejectedState);
                 }
+                
+                ulong playId = playerData.PlayerId(playerName, Convert.ToUInt64(correlationId));
+                if (allowMultipleIds) // Duplicate steam ids will be emulated as seperate players
+                {
+                    foreach (PlayerContext Context in reservations.Values)
```

Comment:
Wouldn't these PlayerContexts need to have their ids sorted for this to work?  If there are three local instances they may be out of order example:

Trying to Connect with: 1 
Already Connected Ids: 2, 1
Output Id: 2 (incorrect)

Or are these always in correct sorted order?

Additionally, it would be ideal to have this whole block in a separate method for readability. Something like, resolveOverlappingSteamIds or something similar.

Reply:
Instead of adding 1 to the steam id maybe use a cpu ticks instead?

`playId += System.Diagnostics.Stopwatch.GetTimestamp();`